test/e2e/run.sh

```diff
@@ -59,8 +68,25 @@ do
   # Some of the tests will modify files in the distribution folder, e.g. cluster test will modify the application.yml
   # so we give each test a separate distribution folder here
   mkdir -p "$test_case" && tar -zxf dist/apache-skywalking-apm-bin.tar.gz -C "$test_case"
-
-  ./mvnw -Dbuild.id="${BUILD_ID:-local}" -De2e.container.version="${E2E_VERSION}" -Dsw.home="${base_dir}/$test_case/apache-skywalking-apm-bin" -f test/e2e/pom.xml -pl "$test_case" -am verify
+  
+   sky_home_dir="${base_dir}/$test_case/apache-skywalking-apm-bin"
+  
+  ./mvnw -Dbuild.id="${BUILD_ID:-local}" -De2e.container.version="${E2E_VERSION}" -Dsw.home="${sky_home_dir}" -f test/e2e/pom.xml -pl "$test_case" -am verify
+    
+  if ${use_mysql} -eq 1; then
+      echo "MySQL database is storage provider..."
+      
+      # Download MySQL connector.
+      curl ${MYSQL_URL} > "${sky_home_dir}/oap-libs/${MYSQL_DRIVER}"
+      [[ $? -ne 0 ]] && echo "Fail to download ${MYSQL_DRIVER}." && exit 1
+      
+      # Modify application.yml to set MySQL as storage provider.
+      cat "${sky_home_dir}/conf/application.yml" | sed '/elasticsearch/,/mysql/d' | sed "/storage:/a \  mysql:" | sed "/storage:/,/receiver-sharing-server:/s/#//" > ${TMP_APP_YML}
+      cat ${TMP_APP_YML} > "${sky_home_dir}/conf/application.yml"
+      rm -f ${TMP_APP_YML}
+  fi
+  
+  ./mvnw -Dbuild.id="${BUILD_ID:-local}" -Dsw.home="${sky_home_dir}" -f test/e2e/pom.xml -pl "$test_case" -am verify
```

Comment:
you missed this `-De2e.container.version="${E2E_VERSION}"`

Reply:
We add this lately, right?

Reply:
> We add this lately, right?

We added this in my last PR, so I meant @tristaZero  missed this when solving the conflicts

Reply:
Could tests be expected failure?

Reply:
Sorry, should i replace Line#89 with Line#74 ?

Reply:
I think you should keep this line unchanged.

Reply:
I am confused about line89 and 74. Why maven run twice?

Reply:
..en.. Can you make sure whether Line#74 is what you want? Maybe i mistaked the Line#74 and #89, only one of which is needed, is it right？

Reply:
> Could tests be expected failure?

Yes definitely

> Sorry, should i replace Line#89 with Line#74 ?

I think so

> I am confused about line89 and 74. Why maven run twice?

Line#74 is my changes in last PR, line#89 is produced by conflict merging? @tristaZero 

> ..en.. Can you make sure whether Line#74 is what you want? Maybe i mistaked the Line#74 and #89, only one of which is needed, is it right？

Line#74 is what I want, refer to the master branch

Reply:
Thanks for your clarification. @kezhenxu94 I fixed it already.